src/components/PopoverListItem/PopoverListItem.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import type { StoryObj, Meta } from '@storybook/react';
+import React from 'react';
+
+import { PopoverListItem } from './PopoverListItem';
+
+export default {
+  title: 'Molecules/Blocks/PopoverListItem',
+  component: PopoverListItem,
+  parameters: {
+    badges: [BADGE.BETA],
+  },
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof PopoverListItem>;
+
+export const Default: StoryObj<Args> = {
+  args: {
+    children: <>Default list item</>,
```

Comment:
nit:
```suggestion
    children: 'Default list item'
```

Reply:
ah i didn't know it would allow strings like that; will update